en-US/Language.xml

```diff
@@ -1097,7 +1263,14 @@ Numbers (move from N entry) -->
 								<row>
 									<entry> Use the utility to run activities and save your settings. </entry>
 								<!-- Split into 2 sentences	 -->
-									 <entry> Depending on the meaning: Use the utility to run activities and to save your settings (if the utility does both functions). Or:  Use the utility to run activities, and then when you are done, save your settings (if the process consists of two separate steps, of which the utility does only one). </entry>
+									 <entry> Depending on the meaning: 
+									 <para>
+									 Use the utility to run activities and to save your settings (if the utility does both functions). 
+									</para>
+									 <para>
+									 <emphasis>Or:</emphasis> Use the utility to run activities, and then when you are done, save your settings (if the process consists of two separate steps, of which the utility does only one). 
```

Comment:
```suggestion
									 <emphasis>Or:</emphasis> Use the utility to run activities, and then when you are done, save your settings (if the process consists of two separate steps, of which the utility does only one). 
```

Reply:
Again, we might make an issue against a future edition. The phrase "when you are done" bit here is extraneous. "...activities, and then save..."

Reply:
Agree and updated in https://github.com/StyleGuides/WritingStyleGuide/pull/449.